run-container.sh

```diff
@@ -45,6 +20,7 @@ usage() {
 	      The -k option takes precedence, overwriting the results of local kubeconfig autodiscovery.
 	      See the 'Kubeconfig lookup order' section below for more details.
 	  -n: set the network mode of the container.
+	  -d: set the DNS resolver address for the test containers started by docker, may be required with certain docker version if the kubeconfig contains host names
```

Comment:
Not sure if we should break the lines ourselves, or depend on the user's terminal emulator to do it for us. So far, we've been (or rather: I've been) breaking the line at some artificially set point, around the 100 character limit.

We could either:
- break the line, like this:
```suggestion
	  -d: set the DNS resolver address for the test containers started by docker, may be required with
	      certain docker version if the kubeconfig contains host names
```
- or, remove newlines from all lines in the usage prompt broken around the character limit, and just depend on word-wrapping.

Reply:
Will break the line